tasks/main.yml

```diff
@@ -65,7 +38,7 @@
     group: 'root'
     mode: '0644'
   when: (item.value.state|d("present") != "absent" and
-         (item.value.type|d("default") not in ["divert"] or item.value.raw|d()))
+         (item.value.type|d("default") not in ["divert", "dont-create"] or item.value.raw|d()))
```

Comment:
I'm leaning towards `ignore` option for this case, see for example `debops.cron` or `debops.ifupdown` role. Seems to better mesh with `present` and `absent`. Perhaps a parameter name change?